src/react/views/about.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import PageLayout from '../templates/page-layout'
+import ArticleGroup from '../components/article-group'
+import ArchivePanel from '../components/panels/archive'
+import SistersPanel from '../components/panels/sisters'
+import ArticleThumb from '../components/article-thumb'
+import FeatureSlider from '../components/feature-slider'
+
+const AboutPage = () =>
+    <PageLayout id="about-page"
+        top={[
+
+        ]}
+
+        main={[
+        <h1 className="strong">Our Mission</h1>,
+          <p className="thin">Our Mission</p>
+
+        ]}
+
+        side={[
+        <div className="row">
+            <div className="col-md-12 col-sm-6">
+                <SistersPanel />
+            </div>
+        </div>
+        ,
+        <div className="row">
+            <div className="col-md-12 col-xs-6">
+                <div className="tile tile-vertical blue-theme">
```

Comment:
Since we use this multiple times lets make a `components/panels/social.js` and then include it here and on the home page